lib/pages/mysite-page.js

```diff
@@ -0,0 +1,29 @@
+// This is the page shown after clicking on the site name from side bar
+
+import { By as by, until } from 'selenium-webdriver';
+import config from 'config';
+
+import BaseContainer from '../base-container.js';
+import * as driverHelper from '../driver-helper.js';
+
+export default class MySitePage extends BaseContainer {
```

Comment:
I would probably call this a `component` over a `page`, and it therefore probably should live under /components.

The main difference is a component isn't accessible by visiting it, so therefore it doesn't the visit parameter and you don't need to determine the URL or anything like that. So the constructor would look something like:

```
	constructor( driver ) {
		super( driver, by.css( '.web-preview__frame' ) );
	}
```

Check out `post-preview-component.js` for inspiration - as this does a very similar thing to this except it previews a post not a site